src/mask.js

```diff
@@ -178,7 +179,7 @@ angular.module('ui.mask', [])
                                 }
                                 iElement.bind('blur', blurHandler);
                                 iElement.bind('mousedown mouseup', mouseDownUpHandler);
-                                iElement.bind('input keyup click focus', eventHandler);
+                                iElement.bind(listOptions.eventsToHandle.join(' '), eventHandler);
```

Comment:
The build is failing because `listOptions` is not defined here. Do you mean `options` (the mask options object)?